README.md

```diff
@@ -7,38 +7,38 @@
 **Kover** - Gradle plugin for Kotlin code coverage tools: [Kover](https://github.com/JetBrains/intellij-coverage)
 and [JaCoCo](https://github.com/jacoco/jacoco).
 
-Minimal supported `Gradle` version: `6.6`.
+Minimum supported version of `Gradle` is `6.6`.
 
 ## Table of contents
+
 - [Features](#features)
 - [Quickstart](#quickstart)
-  - [Apply plugin to a project](#apply-plugin)
-    - [Applying plugins with the plugins DSL](#applying-plugins-with-the-plugins-dsl)
-    - [Legacy Plugin Application: applying plugins with the buildscript block](#legacy-plugin-application-applying-plugins-with-the-buildscript-block)
+  - [Single-project tasks](#single-project-tasks)
   - [Cross-project coverage](#cross-project-coverage)
 - [Configuration](#configuration)
   - [Configuring project](#configuring-project)
   - [Configuring merged reports](#configuring-merged-reports)
-  - [Configuring JVM test task](#configuring-jvm-test-task)
+  - [Configuring JVM test task](#configuring-jvm-test-tasks)
   - [Specifying Coverage Tool](#specifying-coverage-tool)
-- [Kover single-project tasks](#kover-single-project-tasks)
-- [Kover merged tasks](#kover-merged-tasks)
 - [Example of configuring Android application](#example-of-configuring-android-application) 
 - [Implicit plugin dependencies](#implicit-plugin-dependencies)
-- [Building and contributing](#building-and-contributing)
+- [Building locally and contributing](#building-locally-and-contributing)
 
 ## Features
 
-* Collecting code coverage for `JVM` test tasks
-* `XML` and `HTML` report generation
-* Support of `Kotlin/JVM`, `Kotlin Multiplatform` and mixed `Kotlin-Java` sources with zero additional configuration
-* `Kotlin Android` support without the need to divide into build types and flavours
-* Customizable filters for instrumented classes
+* Collection of code coverage through `JVM` test tasks.
+* `HTML` and `XML` reports.
+* Support for `Kotlin/JVM`, `Kotlin Multiplatform` and mixed `Kotlin-Java` sources with zero additional configuration.
+* Support for `Kotlin Android` without the need to divide it into build types and flavours.
+* Verification rules with bounds to keep track of coverage.
+* Customizable filters for instrumented classes.
 
 ## Quickstart
-### Apply plugin
-#### Applying plugins with the plugins DSL
-In top-level build file:
+
+The recommended way of applying Kover is with the 
+[plugins DSL](https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block).
```

Comment:
Decided to stress it since the legacy method has some restrictions when it comes to type-safe DSL